app/src/components/BaseInput.vue

```diff
@@ -1,42 +1,34 @@
 <template>
   <div :class="width">
-    <label :for="id" class="block text-sm font-medium text-gray-700"> {{ label }} </label>
-    <p :for="id" class="text-gray-500 text-sm">{{ description }}</p>
-    <div>
-      <input
-        v-model="val"
-        @input="onInput"
-        :class="[!isValid ? 'border-pink text-pink' : '', customcss ? customcss : '']"
-        :id="id"
-        :name="id"
-        :required="required"
-        :type="type"
-        :readonly="readonly"
-        :disabled="disabled"
-        :placeholder="placeholder"
-      />
-      <div v-if="!isValid" class="absolute inset-y-0 right-0 pr-3 flex items-center pointer-events-none h-12 w-12">
-        <WarningIcon class="icon stroke-pink" />
-      </div>
-      <p v-if="!isValid" class="text-xs text-red-600" :id="`${id}-error`">{{ errorMsg }}</p>
-      <p :for="id" class="text-grey-400 mt-2 text-sm">{{ description }}</p>
-    </div>
+    <input
+      v-model="val"
+      @input="onInput"
+      :class="[!isValid ? '' : '', customcss ? customcss : '']"
```

Comment:
Seems like we don't use the `isValid` check here, so can we make this:
```suggestion
      :class="[customcss ? customcss : '']"
```